src/steps/devices/converter.ts

```diff
@@ -33,6 +33,9 @@ export function createHostAgentEntity(device: Device): Entity {
         firewallEnabled: device['Firewall Enabled'] || undefined,
         fileVaultEnabled: device['FileVault Enabled'] || undefined,
         platform: device['OS Platform'] || undefined,
+        totalMemory: device['Total Memory (GB)'] || undefined,
+        totalDiskSpace: device['Total Disk Space (GB)'] || undefined,
+        lastOnline: device['Last Online'] || undefined,
```

Comment:
`totalMemory` and `totalDiskSpace` are both referenced in the api docs. Is lastOnline present on the response but not mentioned in the docs yet?

Reply:
Yea i appreciate you checking, the docs are not great. Here that property in the response:

![Screen Shot 2022-02-07 at 3 24 30 PM](https://user-images.githubusercontent.com/11580747/152865938-ce7abc03-3677-46ac-88bb-4a87fb2235f2.png)

Reply:
💯  thanks for catching that. WIll fix